src/algebra/module/linear_map.lean

```diff
@@ -637,3 +650,17 @@ end restrict_scalars
 end add_comm_monoid
 
 end linear_equiv
+
+namespace module
+
+/-- In the presence of the module `module.comp_hom S g`, `g : R ≃+* S` is an `R`-linear equiv. -/
```

Comment:
I don't understand this docstring :confused:

Reply:
How about
```suggestion
/-- `g : R ≃+* S` is `R`-linear when the module structure on `S` is `module.comp_hom S g` . -/
```

Reply:
Yeah, that makes more sense. Thanks!